components/tabs/tabs.js

```diff
@@ -1015,6 +997,26 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 		return document.documentElement.getAttribute('dir') === 'rtl';
 	}
 
+	async #tabSelectionHelper(selectedTab) {
+		const selectedPanel = this._getPanel(selectedTab.id);
+		selectedTab.tabIndex = 0;
+
+		await this.updateComplete;
+
+		selectedPanel.selected = true;
+		this._tabs.forEach((tab) => {
```

Comment:
Loops through each tab, if it's not the newly selected one then sets `selected` to false and sets `tabIndex` to -1.